server/api/e2e/gql_document_test.go

```
@@ -0,0 +1,466 @@
+	if err := r.Project.Save(ctx, p); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func TestDocumentOperations(t *testing.T) {
+	srv := httptest.NewServer(documentTestInterceptor(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusNotFound)
+		errMsg := fmt.Sprintf("Path not found: %s (Method: %s)", r.URL.Path, r.Method)
+		w.Write([]byte(errMsg))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)